.storybook/config.js

```diff
@@ -5,6 +5,7 @@ import { withConsole } from '@storybook/addon-console';
 
 import 'typeface-pt-serif/index.css'
 import 'typeface-pt-sans-caption/index.css'
+import '../node_modules/@osu-cass/sb-components/lib/Assets/Styles/bundle.less'
```

Comment:
This can be `import '@osu-cass/sb-components/lib/Assets/Styles/bundle.less'`

Reply:
Right 😅 oops